shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/Mempool.hs

```diff
@@ -190,9 +261,14 @@ overNewEpochState ::
   NewEpochState era ->
   f (NewEpochState era)
 overNewEpochState f st = do
-  res <- f $ mkMempoolState st
-  pure $
-    st
-      { LedgerState.nesEs =
-          (LedgerState.nesEs st) {LedgerState.esLState = res}
-      }
+  f (mkMempoolState st)
+    >$< \(us, ds) ->
+      st
+        { LedgerState.nesEs =
+            (LedgerState.nesEs st)
+              { LedgerState.esLState =
+                  LedgerState.LedgerState us ds
+              }
+        }
+  where
+    (>$<) = flip (<$>)
```

Comment:
:smile:

Reply:
There's already `Data.Functor.<&>` if you're keen on it https://hackage.haskell.org/package/base-4.15.0.0/docs/Data-Functor.html#v:-60--38--62-

Reply:
Aha, I knew it lived somewhere...